README.md

```diff
@@ -1,39 +1,135 @@
-<!-- 
-This README describes the package. If you publish this package to pub.dev,
-this README's contents appear on the landing page for your package.
+# Help Ukraine Widget for Flutter
+This is a port of [Help Ukraine Widget](https://helpukrainewinwidget.org/) -
+this widget helps to let users of your app know how they can help Ukraine
+from anywhere in the world.
 
-For information about how to write a good package README, see the guide for
-[writing package pages](https://dart.dev/guides/libraries/writing-package-pages). 
+### Showcase
 
-For general information about developing packages, see the Dart guide for
-[creating packages](https://dart.dev/guides/libraries/create-library-packages)
-and the Flutter guide for
-[developing packages and plugins](https://flutter.dev/developing-packages). 
--->
+<details open><summary>First widget variation</summary>
 
-TODO: Put a short description of the package here that helps potential users
-know whether this package might be useful for them.
+![](gifs/widget1.gif)
+</details>
+
+<details><summary>Second widget variation</summary>
+    
+![](gifs/widget2.gif)
+</details>
+
+<details><summary>Third widget variation</summary>
+
+![](gifs/widget3.gif)
+</details>
+
+<details><summary>Fourth widget variation</summary>
+
+![](gifs/widget4.gif)
+</details>
 
 ## Features
+By installing this widget in your app, you will help to end the war in Ukraine.
+It will provide links for:
 
-TODO: List what your package can do. Maybe include images, gifs, or videos.
+* [donations](https://uahelp.monobank.ua/),
+* sharing this widget
+* and [other ways to support Ukraine](https://war.ukraine.ua/).
 
 ## Getting started
+Add the package as a dependency to your __pubspec.yaml__:
 
-TODO: List prerequisites and provide or point to information on how to
-start using the package.
+``` yaml
+dependencies:
+    help_ukraine_widget: <latest version>
+```
+
+in your code:
+
+``` dart
+import 'package:help_ukraine_widget/help_ukraine_widget.dart';
+```
 
 ## Usage
 
-TODO: Include short and useful examples for package users. Add longer examples
-to `/example` folder. 
+### Basic usage
+Use `OverlayWidget` to float and align the widget above your app:
+``` dart
+OverlayWidget(
+    // e.g. Alignment.bottomRight
+    alignment: alignment,
+    overlayWidget: HorizontalHelpWidget(),
+    child: MyApp(),
+)
+```
+
+### Customization
+You can also create custom variations of this widget:
+``` dart
+// we used that in our widget, but you do you 
+final order = [HelpWidgetView.collapsed, HelpWidgetView.main, HelpWidgetView.options];
+
+final _controller = TraverseController(order);
 
-```dart
-const like = 'sample';
+HelpWidget(
+    controller: _controller;
+    // view with links to resources that help Ukraine
+    optionsView: ...
+    // transitional view between options and collapsed
+    mainView: ...
```

Comment:
Not really logical to describe it as "transitional" -- this is the view that the users see first as they open the page.

```suggestion
    // The default first view that the users see.
    mainView: ...
```